### DISCUSSION TITLE
session.discarded hook trigger times

### DISCUSSION THREAD

**Discussion by melmalky** [emqx/emqx#13947] [General]
i have two seprate system communcating on EMQX which i can see huge session.discarded hook trigger times ~172866675 , is that something can intriuce freqent disconnection of the mqtt and packets drops?

<img width="602" alt="MQTT discarded" src="https://github.com/user-attachments/assets/ab27f32b-820a-4a9f-a900-c7ca15d17129">

**Comment by thalesmg** [comment]
Hi.
"Discarded" here means that a session was discarded by a second connection that connected using `clean_start = true`.  So it's an expected and normal event once the same clientid disconnects and later reconnects.
